### DISCUSSION TITLE
Question: How can I build dashboard like app in flutter?

### DISCUSSION THREAD

**Discussion by MoathCodes** [mirumee/saleor-dashboard#2773] [Q&A] [answered]
Hi,
I'm trying to build a Dashboard like app with less features in flutter,
 can someone give me a hint on where to start **if it's possible**.
Thank you in advance.

**Comment by kzuraw** [comment, marked as answer]
Hi @IronDragonDev and thank you for creating the issue 👋🏻

I took the liberty and converted it to the discussion - it seems like it is more useful.

Going back to your question - I think the best way to start is with Flutter official [tutorials](https://docs.flutter.dev/reference/tutorials). Saleor is using GraphQL so you will need a Flutter client for sure - something like [graphql_flutter](https://pub.dev/packages/graphql_flutter).

**Reply by MoathCodes** [reply]
Thank you for your time.
I started with graphql, but I think the main thing that is stopping me is, **how can I call admin-level mutations?**, I think I need something like an admin token with absolute permissions.
If such thing exist.

**Reply by manojLondhe** [reply]
**> How can I call admin-level mutations?**

You can create app, set permissions you need, generate token and use same for making API calls for mutations you wish to use https://docs.saleor.io/docs/3.x/dashboard/apps#tokens

**Reply by MoathCodes** [reply]
@manojLondhe Thank you very much this is what I needed.

**Reply by peelar** [reply]
@IronDragonDev Not sure if it helps, but [here](https://github.com/saleor/example-flutter) is an example of a Flutter app communicating with Saleor API.
